_includes/disqus_comments.html

```diff
@@ -3,7 +3,7 @@
   <div id="disqus_thread"></div>
   <script>
     var disqus_config = function () {
-      this.page.url = '{{ page.url | absolute_url }}';
+      this.page.url = '{{ absolute_url }}';
```

Comment:
`absolute_url` is a filter (https://jekyllrb.com/docs/liquid/filters/), and, from my knowledge and testing, it doesn't do anything when used directly.

So, shouldn't this be changed to
```js
this.page.url = '';
```
?

Reply:
I think this did not work for me on GitHub pages.

Reply:
What does `{{ absolute_url }}` expand to, in your case?

I tested, and in my case, it expands to nothing: `this.page.url = '';`, which is what I assumed.

Reply:
You are right. in my case it's `this.page.url = '';`